CONTRIBUTING.md

```diff
@@ -2,10 +2,17 @@
 
 Thank you for your contribution!
 
-## Order
-
-Please ensure `icons_by_filename`, `icons_by_file_extension`, `icons_by_operating_system`, `icons_by_desktop_environment`,
-`icons_by_window_manager` and `filetypes` are ordered alphabetically, to prevent merge conflicts.
+## Quick: Add/Change An Icon
+
+1. Copy or modify a line in `icons-default.lua` and set:
+   - `icon`
+   - `color` (dark)
+   - `name`
+1. Run `make` to:
```

Comment:
My plan is to automate this step on CI. No need to make users do that. What do you think?

Reply:
Would the CI need to commit the changes? That could get complex...

I'd like the contrubitors to do this locally, to encourage them to test first :)

Reply:
Sure, that's fine with me and saves me work. :)